### PR TITLE
Pinned panel updates

### PR DESCRIPTION
**Pull request by ioedeveloper**
Closes https://github.com/ethereum/remix-project/issues/4824

### CODE REVIEW

**Comment by ryestew** [comment]
The cursor should switch to a finger when over the icon to move it to the right or left side.  In a future version there should be a mouseover state of the icon - maybe in bold.  But the tooltip makes it clear that there will be an action happening.  So I will just leave this as a comment for this PR and not a required change.

**Review by ryestew — APPROVED** [review]
This has my approval - but see the note and someone else should approve this as well.

**Review by LianaHus — APPROVED** [review]

**Pull request merged**
